libs/elements/layout/src/lib/pages/elewa-group-main-page/elewa-group-main-page.component.html

```diff
@@ -7,11 +7,75 @@
     <router-outlet></router-outlet>
   </div>
   <div class="footer">
-    <elewa-group-footer>
-      <div footer class="footer-content">
+    <elewa-group-footer [ngSwitch]="route">
+      <div footer class="footer-content" *ngSwitchCase="'/home/en'">
```

Comment:
Switch needs to be inside this class to avoid duplication this div multiple times or switching entire footer component.